src/utils/updateNickname.ts

```diff
@@ -19,10 +19,7 @@ export async function updateNickName(
       body: JSON.stringify(data),
     });
     if (nameChangeResponse.ok) {
-      return await nameChangeResponse.json();
-    } else {
-      console.log(nameChangeResponse.json());
-      return INTERNAL_SERVER_ERROR;
+      return NAME_CHANGED;
     }
   } catch (error) {
     console.log(error);
```

Comment:
Console messages shouldn't be present in prod code

Reply:
as you can see it is a deleted line

Reply:
My bad removing it

Reply:
done